lib/libutils/isoc/include/malloc.h

```diff
@@ -91,33 +83,27 @@ void malloc_reset_stats(void);
 
 #ifdef CFG_NS_VIRTUALIZATION
 
+void *nex_malloc(size_t size);
+void *nex_calloc(size_t nmemb, size_t size);
+void *nex_realloc(void *ptr, size_t size);
+void *nex_memalign(size_t alignment, size_t size);
```

Comment:
These function declaration are now defined even when `ENABLE_MDBG` is disabled. Won't they conflict with the macros defined right below when `ENABLE_MDBG` is enabled?

Reply:
They don't conflict, but they are unnecessary and confusing. I'll change to only define them without `ENABLE_MDBG`.

Reply:
Maybe do the same for `malloc()` and friends.

Reply:
No, those should be kept due to commit eacabbbceb43 ("libutils: ta: provide malloc(), calloc() and realloc() when debug is on")